extensions/azcli/src/azdata.ts

```diff
@@ -262,451 +240,76 @@ export class AzdataTool implements azdataExt.IAzdataApi {
-		default:
-			return await getPlatformReleaseVersion();
-	}
-}
-
-/**
- * Parses out the azdata version from the raw azdata version output
- * @param raw The raw version output from azdata --version
+ * Parses out the Azure CLI version from the raw az version output
+ * @param raw The raw version output from az --version
  */
 function parseVersion(raw: string): string {
 	// Currently the version is a multi-line string that contains other version information such
-	// as the Python installation, with the first line being the version of azdata itself.
-	const lines = raw.split(os.EOL);
-	return lines[0].trim();
+	// as the Python installation, with the first line holding the version of az itself.
+	const start = raw.search('azure-cli');
+	const end = raw.search('core');
+	raw = raw.slice(start, end).replace('azure-cli', '');
+	return raw.trim();
 }
 
 /**
- * Gets the latest azdata version for MacOs clients
+ * Parses out the arcdata extension version from the raw az version output
+ * @param raw The raw version output from az --version
  */
-async function discoverLatestStableAzdataVersionDarwin(): Promise<SemVer> {
-	// set brew tap to azdata-cli repository
-	await executeCommand('brew', ['tap', 'microsoft/azdata-cli-release']);
-	await executeCommand('brew', ['update']);
-	let brewInfoAzdataCliJson;
-	// Get the package version 'info' about 'azdata-cli' from 'brew' as a json object
-	const brewInfoOutput = (await executeCommand('brew', ['info', 'azdata-cli', '--json'])).stdout;
-	try {
-		brewInfoAzdataCliJson = JSON.parse(brewInfoOutput);
-	} catch (e) {
-		throw Error(`failed to parse the JSON contents output of: 'brew info azdata-cli --json', text being parsed: '${brewInfoOutput}', error:${getErrorMessage(e)}`);
+function parseArcExtensionVersion(raw: string): string {
+	// Currently the version is a multi-line string that contains other version information such
+	// as the Python installation and any extensions.
+	const start = raw.search('arcdata');
+	if (start === -1) {
+		vscode.window.showErrorMessage(loc.arcdataExtensionNotInstalled);
+	} else {
+		raw = raw.slice(start + 7);
+		raw = raw.split(os.EOL)[0].trim();
 	}
-	// Get the 'info' about 'azdata-cli' from 'brew' as a json object
-	const azdataPackageVersionInfo: AzdataDarwinPackageVersionInfo = brewInfoAzdataCliJson.shift();
-	Logger.log(loc.latestAzdataVersionAvailable(azdataPackageVersionInfo.versions.stable));
-	return new SemVer(azdataPackageVersionInfo.versions.stable);
+	return raw.trim();
 }
 
-async function executeAzdataCommand(command: string, args: string[], additionalEnvVars: azdataExt.AdditionalEnvVars = {}): Promise<ProcessOutput> {
+async function executeAzCommand(command: string, args: string[], additionalEnvVars: azExt.AdditionalEnvVars = {}): Promise<ProcessOutput> {
 	additionalEnvVars = Object.assign(additionalEnvVars, { 'ACCEPT_EULA': 'yes' });
```

Comment:
You should chat with Sean about setting this value ourselves - typically we should only be setting that if we've asked the user to accept the EULA and they have said yes (we can't just assume they accept it). But you're getting rid of the EULA prompt stuff right?

Reply:
Will do, started that conversation with Sean